diff --git a/node_modules/@gorhom/bottom-sheet/src/hooks/useGestureEventsHandlersDefault.tsx b/node_modules/@gorhom/bottom-sheet/src/hooks/useGestureEventsHandlersDefault.tsx
index 0ded828..daaca30 100644
--- a/node_modules/@gorhom/bottom-sheet/src/hooks/useGestureEventsHandlersDefault.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/hooks/useGestureEventsHandlersDefault.tsx
@@ -65,6 +65,10 @@ export const useGestureEventsHandlersDefault: GestureEventsHandlersHookType =
     });
     //#endregion
 
+    const dismissKeyboard = () => {
+      Keyboard.dismiss();
+    };
+
     //#region gesture methods
     const handleOnStart: GestureEventHandlerCallbackType = useWorkletCallback(
       function handleOnStart(__, _) {
@@ -318,7 +322,7 @@ export const useGestureEventsHandlersDefault: GestureEventsHandlersHookType =
               absoluteY > WINDOW_HEIGHT - animatedKeyboardHeight.value
             )
           ) {
-            runOnJS(Keyboard.dismiss)();
+            runOnJS(dismissKeyboard)();
           }
         }
 
